packages/kmod-6.1-nvidia-r570/kmod-6.1-nvidia-r570.spec

```diff
@@ -515,6 +532,14 @@ popd
 # systemd units
 %{_cross_unitdir}/nvidia-persistenced.service
 
+# ICD / vendor descriptors
+%{_cross_datadir}/vulkan/icd.d/nvidia_icd.json
+%{_cross_datadir}/vulkan/icd.d/nvidia_layers.json
+ln -s %{_cross_datadir}/vulkan/icd.d/nvidia_layers.json %{_cross_datadir}/vulkan/implicit_layer.d/nvidia_layers.json
```

Comment:
This isn't quite correct. Symlinking should be done in the `%install` section like so:

```spec
ln -rs %{buildroot}%{_cross_datadir}/vulkan/icd.d/nvidia_layers.json %{buildroot}%{_cross_datadir}/vulkan/implicit_layer.d/nvidia_layers.json
```

And the same in the other specs.

Reply:
Hey @iterion, we are happy to take it from here if you don't have the cycles to complete this last small change!

Reply:
Yes, please! Working on a launch this week so I'm swamped. I'll have time next week probably but please don't wait on me!